tools/uf2conv.py

```diff
@@ -317,18 +317,23 @@ def error(msg):
 
     if args.serial:
         if str(args.serial).startswith("/dev/tty") or str(args.serial).startswith("COM") or str(args.serial).startswith("/dev/cu"):
-            try:
-                print("Resetting "+str(args.serial))
+            print("Entering serial port reset")
+            for tries in range(5):
                 try:
-                    ser = serial.Serial(args.serial, 1200)
-                    ser.dtr = False
+                    print("Resetting " + str(args.serial) + " try " + str(tries))
```

Comment:
This makes it take ~2 minutes to upload whenever you use the BOOTSEL to make it upload.  I'm not sure what retrying the serial (which should be steady available anyway for a long time before the upload process) is really buying you, but that;s just too painful.

Reply:
OK. I'll revert that out.